spec.bs

```diff
@@ -1522,6 +1533,23 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. Return |dummyURL|'s [=url/host=].
 </div>
 
+<div algorithm>
+  To <dfn>canonicalize an IPv6 hostname</dfn> given a string |value|:
+
+  1. Let |result| be the empty string.
+  1. [=list/For each=] |code point| in |value| interpreted as a sequence of [=/code points=]:
```

Comment:
I like this a lot.

Reply:
As per chat, changing "sequence" to "[=list=]".